code-challenges.test.js

```diff
@@ -1,52 +1,165 @@
+    
+    console.log(divideBy3(object1))
+    console.log(divideBy3(object2))
+    console.log(divideBy3(object3))
+
+// //     // Spent WAY WAY to much time on this problem.. Test Failed!
+// //     // cannot get the expected output - Receiving reference errors. Tried different ways to manipulate the code.
+// //     // I thought perhaps the function itself needed to be within the object culy brackets 
+// //     // Shown below:
+
+    const object1 = { number: 15,
+        
+        divideBy3: function() {
+        
+            if ((this.number) % 3 === 0) {
+                `${this.number} "is divisible by three`}
+            else if ((this.number) % 3 !== 0) {
+                `${this.number} "is not divisible by three`}
+        }
+    }
+        Console.log(object1)
+
+// // // This way gave me the same referencing errors
+// // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
+
+// // a) Create a test with expect statements for each of the variables provided.
 
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
+describe("randomNouns1", () => {
+    it("returns an array with all the words capitalized", () => {
+      expect(randomNouns1()).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+
+describe("randomNouns1", () => {
+    it("returns an array with all the words capitalized", () => {
+        expect(randomNouns1()).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+
+// // Good Error
+
 // b) Create the function that makes the test pass.
 
+const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+// // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+
+// Psuedocode
+// Create a function that takes in an array of words and returns an array with all the words capitalized.
+// Input: randomNouns1, randomNouns2
+// Output: Capitalizes the each word in the array set
+//  Use built in methods .toUpperCase and .substring
+
+const capitalLetter = (array) => {
+    return array.map((value) => {
+        return value[0].toUpperCase() + value.substring(1)
+    })
+
+}
+console.log(capitalLetter(randomNouns1))
+console.log(capitalLetter(randomNouns2))
+
+// could not get jest to pass - following error
+
+// FAIL  ./code-challenges.test.js
+// ● Test suite failed to run
+
+//   Jest encountered an unexpected token
+
+//   Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.
```

Comment:
Correct, it looks like there are some closing parentheses and curly braces missing to close off your tests.  Use resources to help write out the syntax for jest (it is definitely weird and easy to miss something small).